app/src/main/java/org/dosomething/letsdothis/ui/fragments/ActionsFragment.java

```diff
@@ -43,6 +45,8 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
     {
         super.onViewCreated(view, savedInstanceState);
         TabPageIndicator indicator = (TabPageIndicator) view.findViewById(R.id.indicator);
+        int pxFromDip = ViewUtils.getPxFromDip(getResources(), INDICATOR_SPACING).intValue();
```

Comment:
For this you can just do getResources().getDimensionPixelSize(R.dimen.padding_small)